### ISSUE TITLE
An input prompt for emojis

### ISSUE THREAD

**Issue by ikariiin** [SaitamaSama/Centaur#8]

**Comment by pandaa880** [comment]
Is it using React for FE ?

**Comment by pandaa880** [comment]
It doesn't run on my local machine. I started both server but it is stuck on **starting**.
Even if I refresh, nothing happens.

**Comment by ikariiin** [comment]
@pandaa880 Yeah, well I pushed something to make it work with the dev build. And should probably work fine now. For how it works atm you can see it in action here https://centaur.xbenjii.co.uk And it uses plain ol' simple React for client side rendering.
